chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -53,7 +53,7 @@
   action :add_tenant
 end.run_action(:add_tenant)
 
-%x{keystone --os_username #{tempest_comp_user} --os_password #{tempest_comp_pass} --os_tenant_name #{tempest_comp_tenant} --os_auth_url #{keystone_settings["internal_auth_url"]} endpoint-get --service orchestration &> /dev/null}
+%x{keystone --insecure --os_username #{tempest_comp_user} --os_password #{tempest_comp_pass} --os_tenant_name #{tempest_comp_tenant} --os_auth_url #{keystone_settings["internal_auth_url"]} endpoint-get --service orchestration &> /dev/null}
```

Comment:
Well, I think I disagree with that statement -- it's also a way to test the CLI to make sure that valid certificates are indeed valid according to the CLI.

And it will create issues if the code is copied & pasted elsewhere.